nextcord/ext/dpy/slash.py

```diff
@@ -0,0 +1,44 @@
+"""
+The MIT License (MIT)
+
+Copyright (c) 2021-present tag-epic
+
+Permission is hereby granted, free of charge, to any person obtaining a
+copy of this software and associated documentation files (the "Software"),
+to deal in the Software without restriction, including without limitation
+the rights to use, copy, modify, merge, publish, distribute, sublicense,
+and/or sell copies of the Software, and to permit persons to whom the
+Software is furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+DEALINGS IN THE SOFTWARE.
+"""
+
+
+from nextcord.application_command import SlashApplicationCommand, SlashApplicationSubcommand, CallbackWrapper
+from typing import Union
+
+
+def describe(**kwargs: str) -> Union[SlashApplicationCommand, SlashApplicationSubcommand, "DescribeWrapper"]:
```

Comment:
Is this necessary as we already have 2 ways of describing options?

Also, why does it need to be in ext?

Reply:
Not necessary, it was created because the Discord.py way of setting descriptions happened to be a perfectly easy way to test `CallbackWrapper`. It's in an ext because it doesn't belong in the library, in the `dpy` ext specifically because I found it funny. I will remove it if you don't approve of it.

Reply:
Renaming this namespace to slash_utils as an ext. This will continue to exist, but as a less-jokey example of how to use CallbackWrapper.